### ISSUE TITLE
testing script

### ISSUE THREAD

**Issue by adipriyantobpn** [yeesoft/yii2-yee-cms#25]
Any plan to provide testing script for each module?

**Comment by yeesoft** [comment]
Yes, but not in closest future. Unfortunately I haven't enough time to do this. In addition YeeCMS in under active development and a lot of things may change at any moment. First of all I am going to release 0.1.0 version, without testing scripts.
It would be great if someone help me with testing script.

**Comment by adipriyantobpn** [comment]
i'm happy to help :)

On Wed, May 18, 2016 at 8:55 PM, YeeSoft <notifications@github.com> wrote:

> Yes, but not in closest future. Unfortunately I haven't enough time to do
> this. In addition YeeCMS in under active development and a lot of things
> may change at any moment. First of all I am going to release 0.1.0 version,
> without testing scripts.
> It would be great if someone help me with testing script.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/yeesoft/yii2-yee-cms/issues/25#issuecomment-220033336>
>



-- 
Best Regards,

Adi Priyanto

**Comment by yeesoft** [comment]
That's great! Good news for me! I'm waiting for pull requests from you. Thanks a lot!
